packages/@react-spectrum/searchfield/docs/SearchField.mdx

```diff
@@ -171,6 +171,30 @@ By default, the label is positioned above the SearchField. The `labelPosition` p
 <SearchField label="Search" labelPosition="side" labelAlign="end" />
 ```
 
+### Help text
+[View guidelines](https://spectrum.adobe.com/page/help-text/#Usage-guidelines)
```

Comment:
I noticed their docs show that for error messages and the API has 'showErrorIcon`. I think one of these examples should have that too.

Reply:
correct